components/styles/interval.css

```diff
@@ -0,0 +1,33 @@
+.control-panel-interval-{{ UUID }} {
+  -webkit-appearance: none;
+  position: absolute;
+  height: 20px;
+  margin: 0px 0;
+  width: 33%;
+  left: 52.5%;
+  background-color: {{ TRACK_COLOR }};
+  cursor: ew-resize;
+
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+.control-panel-interval-{{ UUID }} .control-panel-interval-handle {
+  background-color: {{ INTERVAL_COLOR }};
+  position: absolute;
+  height: 20px;
+  min-width: 1px;
+}
+#control-panel{{ UUID }}.control-panel-interval-dragging * {
```

Comment:
should there be a hyphen here for consistency with the others e.g. `#control-panel-{{ UUID }}`? might need another change elsewhere